augur/augur_client.py

```diff
@@ -57,17 +57,20 @@ def __init__(self, hostname, port):
       response_data['lastTradeTime'] = (
         self._ensure_unix_timestamp(response_data['lastTradeTime']))
     # Ensure decimals default to None if needed.
-    response_data['initialReportSize'] = (
-      response_data['initialReportSize']
-      if response_data['initialReportSize'] else None)
-    response_data['finalizationBlockNumber'] = (
-      response_data['finalizationBlockNumber']
-      if response_data['finalizationBlockNumber'] else None)
-    response_data['lastTradeBlockNumber'] = (
-      response_data['lastTradeBlockNumber']
-      if response_data['lastTradeBlockNumber'] else None)
+    if response_data['initialReportSize']:
+      response_data['initialReportSize'] = (
+        Decimal(response_data['initialReportSize']))
+    if response_data['finalizationBlockNumber']:
```

Comment:
The API states that some of these are ```int```, not ```Decimal```. Review every single parameter very carefully to ensure it's initialized properly.